src/Console/Command/FixCommand.php

```diff
@@ -212,6 +212,7 @@ protected function configure(): void
                 new InputOption('dry-run', '', InputOption::VALUE_NONE, 'Only shows which files would have been modified.'),
                 new InputOption('rules', '', InputOption::VALUE_REQUIRED, 'List of rules that should be run against configured paths.'),
                 new InputOption('using-cache', '', InputOption::VALUE_REQUIRED, 'Should cache be used (can be `yes` or `no`).'),
+                new InputOption('fail-on-unsupported-version', '', InputOption::VALUE_REQUIRED, 'Should the command refuse to run on unsupported PHP version (can be `yes` or `no`).'),
```

Comment:
version of what? PHP CS Fixer?

we have `allow-risky`. let's go with `allow-not-yet-supported-php`

Reply:
@keradus OK, since the default was changed (which was to accept unsupported versions by default and need to opt-in to fail), makes sense we change the name here.

`not-yet-supported` seems weird, I suggest `allow-unsupported-php-version`